### PR TITLE
Update endpoint-dlp-getting-started.md

### PR DESCRIPTION
**Pull request by MasLibman**
I suggest we optimize towards mentioning windows 10 / 11 when setting context and in requirements and then leaving it at "Windows".

I dont think onboarding needs to mention Windows version unless there is a specific different for onboarding based upon Windows version. 
@chrfox

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 1]
```diff
@@ -72,18 +72,18 @@ Data from Endpoint DLP can be viewed in [Activity explorer](data-classification-
 - Security admin
 - Compliance data admin
 
-### Prepare your Windows 10 endpoints
+### Prepare your Windows 10/11 endpoints
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:76]:
```suggestion
### Prepare your Windows 10/Windows 11 endpoints
```

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 2]
```diff
@@ -72,18 +72,18 @@ Data from Endpoint DLP can be viewed in [Activity explorer](data-classification-
...
-1. Must be running Windows 10 x64 build 1809 or later.
+1. Must be running Windows 10 x64 build 1809, Windows 11, or later.
 
 1. Antimalware Client Version is 4.18.2009.7 or newer. Check your current version by opening Windows Security app, select the Settings icon, and then select About. The version number is listed under Antimalware Client Version. Update to the latest Antimalware Client Version by installing Windows Update KB4052623.
 
    > [!NOTE]
    > None of Windows Security components need to be active, you can run Endpoint DLP independent of Windows Security status, but the [Real-time protection and Behavior monitor](/windows/security/threat-protection/microsoft-defender-antivirus/configure-real-time-protection-microsoft-defender-antivirus)) must be enabled.
 
-1. The following Windows Updates are installed.
+1. The following Updates are installed on Windows 10 devices
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md:87]:
```suggestion
1. The following updates are installed on Windows 10 devices
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @MasLibman, kindly commit the minor suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
@MasLibman Please implement the changes, if you agree. Thanks

**Pull request merged**